jobs/update-colin-filings/pyproject.toml

```diff
@@ -0,0 +1,50 @@
+[tool.poetry]
+name = "update-colin-filings"
+version = "0.1.0"
+description = "Update Colin filings job"
+authors = ["Hongjing Chen <Hongjing.Chen@gov.bc.ca>"]
+license = "BSD-3-Clause"
+
+[tool.poetry.dependencies]
+python = "^3.10"
+Flask = "2.3.2"
+Flask-Moment = "^1.0.5"
+Flask-Script = "^2.0.6"
+Jinja2 = "^3.1.2"
+MarkupSafe = "^2.1.3"
+Werkzeug = "^2.3.7"
+aniso8601 = "^9.0.1"
+attrs = "^23.1.0"
+blinker = "^1.6.2"
+certifi = "^2023.7.22"
+chardet = "^5.2.0"
+click = "^8.1.7"
+ecdsa = "^0.18.0"
+flask-jwt-oidc = "^0.3.0"
+flask-restplus = "^0.13.0"
+gunicorn = "^20.1.0"
+idna = "^3.4"
+itsdangerous = "^2.1.2"
+jsonschema = "^4.17.3"
+pyasn1 = "^0.5.0"
+pyrsistent = "^0.19.3"
+python-dotenv = "^1.0.0"
+python-jose = "^3.3.0"
+pytz = "^2023.3.post1"
+requests = "^2.31.0"
+rsa = "^4.9"
+sentry-sdk = "^1.30.0"
+six = "^1.16.0"
+urllib3 = "^1.22.0"
+legal_api = { git = "https://github.com/bcgov/lear.git", subdirectory = "legal-api", branch = "dev_legal_name_changes" }
```

Comment:
It cannot move to use only business-model because update colin filings job refers to `legal_api.services`.